external-api/src/v1/controllers/party-db.controller.ts

```diff
@@ -21,11 +21,40 @@ export const lookup = async (req: Request, res: Response) => {
 
   const response: { status: number; data: unknown } = await axios({
     method: 'get',
-    url: `${APIM_MDM_URL}customers?companyReg=${companyReg}`,
+    url: AUTOMATIC_SF_CUSTOMER_CREATION_ENABLED === 'true'
+      ? `${APIM_MDM_URL}customers/direct?companyRegistrationNumber=${companyReg}`
+      : `${APIM_MDM_URL}customers?companyReg=${companyReg}`,
     headers,
   }).catch((error: AxiosError) => {
     console.error('Error calling Party DB API %o', error);
-    return { data: 'Failed to call Party DB API', status: error?.response?.status || HttpStatusCode.InternalServerError };
+    return { data: 'Error calling Party DB API', status: error?.response?.status || HttpStatusCode.InternalServerError };
+  });
+
+  const { status, data } = response;
+
+  return res.status(status).send(data);
+};
+
+export const createParty = async (req: CustomExpressRequest<{ reqBody: { companyName: string } }>, res: Response) => {
```

Comment:
aware the testing in this area is not great atm so you might have to set a couple of things up yourself, but testing different inputs and the validation of company registration number etc. would be good here!